### DISCUSSION TITLE
General Question: When can i expect the next lawnchair release

### DISCUSSION THREAD

**Discussion by notroger7** [LawnchairLauncher/lawnchair#3048] [Q&A] [answered]
Hello, i tried asking in telegram chat but didn't get the response I needed, so may I ask the devs, when can I expect the next lawnchair release?

**Comment by SuperDragonXD** [comment, marked as answer]
Hey there! We really don't specifically know when the next Lawnchair will release (unlike Lawnicons), since we can only estimate by how many new notable features, bug fixes, and etc. Most of the commits of current Lawnchair have been mostly about QoL improvements and minor tweaks, not necessarily new features the public can enjoy.

Alongside that, we're still waiting for Paphomb to not be busy (he's in university iirc) so that he can rebase and potentially release Lawnchair 13. We haven't decided yet whether to pospone releasing until L13 arrives, or just release L12.1 Alpha 5 now.

**Reply by TilloX** [reply]
That has been the case for last 3 years now🥲.

**Reply by notroger7** [reply]
I think that now it's good to put more improvements and bug fixes and release 12.1 A5 cuz I compared both A4 and the latest dev build and it looks way different...anyways thanks for the reply
